addons/vehicleDamage/functions/fnc_processHit.sqf

```diff
@@ -0,0 +1,319 @@
+#include "script_component.hpp"
+/*
+ * Author: Brandon (TCVM)
+ * Process hit by projectile against vehicle and apply appropiate damage to part
+ *
+ * Arguments:
+ * 0: The vehicle
+ * 1: Projectile that hit
+ * 2: Hit index of potentially damaged part
+ * 3: New damage done to part
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [myVehicle, projectile, 5, 0.663] call ace_vehicleDamage_fnc_processHit;
+ *
+ *
+ * Public: No
+ */
+params ["_vehicle", "_projectile", "_hitIndex", "_newDamage", "_hitpointData", "_injurer"];
+_hitpointData params ["_hitArea", "_hitpointConfig", "_hitpointName"];
+
+private _return = true;
+
+private _currentPartDamage = _vehicle getHitIndex _hitIndex;
+private _nextPartDamage = _currentPartDamage + _newDamage;
+
+// damage is high enough immediate KO
```

Comment:
```suggestion
// damage is high enough for immediate KO
```
But, you talk about KO == unconscious, but what you are really doing here is killing people.

Reply:
KO = knock-out which in common usage can either mean unconscious or destroy. I'll change it for clarity